mp/src/game/shared/momentum/mom_system_gamemode.cpp

```diff
@@ -51,6 +51,13 @@ void CGameModeBase::OnPlayerSpawn(CMomentumPlayer *pPlayer)
 #endif
 }
 
+void CGameModeBase::ExecGameModeCfg()
+{
+#ifdef CLIENT_DLL
```

Comment:
Not needed. You already do the check when calling this function.

Reply:
Without this, the game fails to build

Reply:
Fair enough